app/assets/javascripts/components/student/student_contact.es6.jsx

```diff
@@ -31,16 +31,16 @@ class StudentContact extends Component {
     return (
         <div style={this.styles.container}>
           <h2 style={this.styles.heading}>{'Contact'}</h2>
```

Comment:
Could you try removing the `this.styles.heading` style here and changing the `h2` tag to an `h4`?